tests/functions.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+function wait_for_resource() {
+  local namespace="$1"
+  local time="$2"
+  local resource="$3"
+  
+  local substring="AGE"
+
+  for i in $(seq 0 $time)
+  do
+    get="$(kubectl get -n $namespace $resource)"
+    if [[ ! $get =~ $substring ]]; then
+      echo "Waiting for $resource in $i iteration"
+      sleep 1
```

Comment:
kubectl get returns (`$?`) 1 if there is no such resource. It should be used instead of string comparison